setup.cfg

```diff
@@ -39,7 +39,7 @@ packages = find:
 include_package_data = true
 zip_safe = false
 install_requires =
-    asgiref >= 3.5.2
+    asgiref >= 3.6.0
```

Comment:
Please add a release note and update docs (see e.g. 2a2bde52f31e09e95ce616e8e6bc0ffeb68f76c9).

Reply:
Yep, good. Thanks. 👍